src/api/libcellml/analyser.h

```diff
@@ -31,10 +31,10 @@ namespace libcellml {
 class LIBCELLML_EXPORT Analyser: public Logger
 {
 public:
-    ~Analyser() override; /**< Destructor. */
-    Analyser(const Analyser &rhs) = delete; /**< Copy constructor. */
-    Analyser(Analyser &&rhs) noexcept = delete; /**< Move constructor. */
-    Analyser &operator=(Analyser rhs) = delete; /**< Assignment operator. */
+    ~Analyser() override; /**< Destructor, @private. */
+    Analyser(const Analyser &rhs) = delete; /**< Copy constructor, @private. */
+    Analyser(Analyser &&rhs) noexcept = delete; /**< Move constructor, @private. */
+    Analyser &operator=(Analyser rhs) = delete; /**< Assignment operator, @private. */
```

Comment:
Why `@private` since those methods are `public`? The same for (all the?) other classes whether Analyser-related or not.

Reply:
I believe the `@private` is there to encourage doxygen to produce documentation that is useful (since these are public methods that shouldn't appear in the documented API), and maybe also hints to be used as part of the JS binding work...

Reply:
Is that the case @hsorby? If so, the I wouldn't be impressed, but whatever works for us, I guess.

Reply:
Yes I am using it to tidy up the Doxygen output.  I am open for alternatives but this is the best compromise that I have at the moment.

Reply:
I don't know enough about Doxygen to suggest an alternative (assuming there is one!). So, "happy" for the current status quo.